### DISCUSSION TITLE
Welcome to A-Lab Discussions!

### DISCUSSION THREAD

**Discussion by jmrplens** [jmrplens/A-Lab#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mwp4444** [comment]
I am a Ph.D. student in mechanical engineering. We are going to measure noise reduction based on aerogel developed by us during nanocomposite material research. I feel a lot of gratitude for A-Lab. Thanks to this, it seems that noise reduction research can be conducted. I have an inquiry during the measurement.

Based on the impedance tube, sound absorption was measured, and the measured value was negative. The sample file you attached also has a negative number.
It is known that a value between 0 and 1 should be obtained for the sound options coefficient value. (Refer to the paper)
Matlab determines that modification is necessary mathematically, and I am also looking at the code from the beginning.
Please tell me if I'm going in the wrong direction.
[Measurement_of_the_Acoustic_Absorption_Coefficient.pdf](https://github.com/jmrplens/A-Lab/files/7125149/Measurement_of_the_Acoustic_Absorption_Coefficient.pdf)
